app/authentication/auth.py

```diff
@@ -127,4 +136,4 @@ def handle_admin_key(auth_token, secret):
     except TokenExpiredError:
         raise AuthError("Invalid token: expired, check that your system clock is accurate", 403)
     except TokenDecodeError:
-        raise AuthError("Invalid token: signature, api token is not valid", 403)
+        raise AuthError("Invalid token: API token is not valid", 403)
```

Comment:
Pointing them to the docs don't quite make sense here as this is an admin endpoint (ie us or one of the apps). We could just keep this short and sweet - `Invalid token` or `Invalid token: Could not decode your token`

Reply:
👍 
What about `Invalid token: Could not decode your API token`?